src/ocean/sys/EventFD.d

```diff
@@ -118,7 +121,18 @@ import core.sys.posix.unistd: read, write, close;
 
 *******************************************************************************/
 
-private extern ( C ) int eventfd ( uint initval, int flags );
+extern ( C )
+{
+    private int eventfd ( uint initval, EFD flags );
```

Comment:
Oh, I found a couple of more places like this where functions are declared that are/should/will soon be in Tango. I would leave this cleanup to a separate PR.